### PR TITLE
Remove unicode characters since everything else is ascii

### LINKED ISSUE #173: Unicode Errors (System specific)
### Actual Behavior

Traceback (most recent call last):
  File "pokecli.py", line 347, in <module>
    main()
  File "pokecli.py", line 340, in main
    bot.run()
  File "OpenPoGoBot/pokemongo_bot/**init**.py", line 92, in run
    self.navigator.navigate(map_cells)
  File "OpenPoGoBot/pokemongo_bot/navigation/fort_navigator.py", line 40, in navigate
    logger.log("[#] Walking towards PokeStop \"{}\" ({} away)".format(fort_name, format_dist(dist, unit)))
UnicodeEncodeError: 'ascii' codec can't encode character u'\xe2' in position 24: ordinal not in range(128)
### Other Information

OS: debian
Git Commit: 7df83ac094901222febaf5c34620e86b3dd34ffa

### PR DESCRIPTION
**Pull request by Reaver01**
### Changes:
- replaces unicode chars at the api wrapper level until we can process all strings as unicode.

Closes #173

@OpenPoGo/maintainers

### CODE REVIEW

**Comment by Reaver01** [comment]
Don't merge yet

**Comment by Reaver01** [comment]
wait to merge, still doesn't fix

**Comment by Reaver01** [comment]
Fixes it on FreeBSD. Still needs testing on Windows

**Comment by kometes-** [comment]
[2016-08-02 00:16:32] [#] Walking Finished
[2016-08-02 00:16:35] [#] Now at PokeStop "Indian Trace Fountain"
Traceback (most recent call last):
  File "pokecli.py", line 347, in <module>
    main()
  File "pokecli.py", line 340, in main
    bot.run()
  File "/home/me/pokeapi/OpenPoGoBot/pokemongo_bot/**init**.py", line 92, in run
    self.navigator.navigate(map_cells)
  File "/home/me/pokeapi/OpenPoGoBot/pokemongo_bot/navigation/fort_navigator.py", line 40, in navigate
    logger.log("[#] Walking towards PokeStop \"{}\" ({} away)".format(fort_name, format_dist(dist, unit)))
UnicodeEncodeError: 'ascii' codec can't encode character u'\xe0' in position 7: ordinal not in range(128)

**Comment by Reaver01** [comment]
@kometes is that with this patch?

**Comment by Reaver01** [comment]
Please run python and then the following:  

```
import platform  
platform.system()  
```

and paste the output.

**Comment by kometes-** [comment]
CYGWIN_NT-6.1-WOW

**Comment by kometes-** [comment]
The 64-bit version of Cygwin returns "CYGWIN_NT-6.1"

**Comment by Reaver01** [comment]
Try this now I just added that to it.

**Comment by Reaver01** [comment]
You are correct. that isn't getting to the logger.  is that the only place you run into the errors?

**Comment by Reaver01** [comment]
@kometes- Can you see if this works?

**Comment by Reaver01** [comment]
I will test it on FreeBSD when I get home.

**Comment by kometes-** [comment]
Also need this fixed:

```
--- a/plugins/spin_pokestop/__init__.py
+++ b/plugins/spin_pokestop/__init__.py
@@ -68,7 +68,7 @@ def spin_pokestop(bot, pokestop=None):
                                                 latitude=pokestop.latitude,
                                                 longitude=pokestop.longitude).call()
     dist = distance(bot.stepper.current_lat, bot.stepper.current_lng, pokestop.latitude, pokestop.longitude)
-    log("Nearby PokeStop found \"{}\" ({} away)".format(fort_details["fort"].fort_name,
+    log("Nearby PokeStop found \"{}\" ({} away)".format(repr(fort_details["fort"].fort_name),
                                                          format_dist(dist, bot.config.distance_unit)), color="yellow")

     log("Spinning...", color="yellow")
```

And can you use platform.system()[:6] == "CYGWIN" so I can use it on both of my systems (32- and 64-bit)?

**Comment by Reaver01** [comment]
that should do it.

File: plugins/spin_pokestop/__init__.py [hunk 1]
```diff
@@ -68,7 +69,11 @@ def log(text, color="black"):
                                                 latitude=pokestop.latitude,
                                                 longitude=pokestop.longitude).call()
     dist = distance(bot.stepper.current_lat, bot.stepper.current_lng, pokestop.latitude, pokestop.longitude)
-    log("Nearby PokeStop found \"{}\" ({} away)".format(fort_details["fort"].fort_name,
+    if platform.system() == "Windows" or platform.system() == "FreeBSD" or platform.system()[:6] == "CYGWIN":
+        fort_name = repr(fort_details["fort"].fort_name)
+    else
```
Comment (Saturate) [inline plugins/spin_pokestop/__init__.py]: You are missing a `:` after `else`, I get an error: 

```
Traceback (most recent call last):
  File ".\pokecli.py", line 347, in <module>
    main()
  File ".\pokecli.py", line 335, in main
    bot.start()
  File "C:\Users\OpenPoGoBot\pokemongo_bot\__init__.py", line 69, in start
    self._init_plugins()
  File "C:\Users\ekalkj\Documents\GitHub\OpenPoGoBot\pokemongo_bot\__init__.py", line 58, in _init_plugins
    self.plugin_manager.load_plugin(plugin)
  File "C:\Users\ekalkj\Documents\GitHub\OpenPoGoBot\pokemongo_bot\plugins.py", line 60, in load_plugin
    module = imp.load_module(self.main_module, *plugins[plugin_name]['info'])
  File "./plugins\spin_pokestop\__init__.py", line 74
    else
       ^
SyntaxError: invalid syntax
```

**Comment by Saturate** [comment]
On windows i get a syntax error:

```
Traceback (most recent call last):
  File ".\pokecli.py", line 347, in <module>
    main()
  File ".\pokecli.py", line 335, in main
    bot.start()
  File "C:\OpenPoGoBot\pokemongo_bot\__init__.py", line 69, in start
    self._init_plugins()
  File "C:\OpenPoGoBot\pokemongo_bot\__init__.py", line 58, in _init_plugins
    self.plugin_manager.load_plugin(plugin)
  File "C:\OpenPoGoBot\pokemongo_bot\plugins.py", line 60, in load_plugin
    module = imp.load_module(self.main_module, *plugins[plugin_name]['info'])
  File "./plugins\spin_pokestop\__init__.py", line 74
    else
       ^
SyntaxError: invalid syntax
```

**Comment by Reaver01** [comment]
Fixed.

**Comment by Saturate** [comment]
Tested and works ~~perfect~~ now :+1:

**Comment by Reaver01** [comment]
Woot! I will test on my FreeBSD system when I get home!

**Comment by Saturate** [comment]
Oh I was a little too quick, it did not break anything, but the chars are not rendered correctly in the console.

```
[2016-08-02 09:44:44] [#] Now at PokeStop "'Gl. M\xf8nt T\xe5rn'"
[2016-08-02 09:44:45] [#] Walking towards PokeStop "'Sko V\xe6gen'" (0.34km away)
```

**Comment by Reaver01** [comment]
Correct.

**Comment by GucciChen** [comment]
i tested ，dosen't work ，my system is kali-rolling .
also
UnicodeEncodeError: 'ascii' codec can't encode character u'\u2013' in position 11: ordinal not in range(128)

**Comment by Reaver01** [comment]
Please test https://github.com/OpenPoGo/OpenPoGoBot/pull/177

**Comment by Reaver01** [comment]
This is too messy and #177 invalidates it. Closing

**Comment by Reaver01** [comment]
Pay no attention to this for now.

**Comment by Reaver01** [comment]
Good to merge.

**Comment by Reaver01** [comment]
@OpenPoGo/maintainers Please merge this

**Comment by MaxLeiter** [comment]
:+1:  from me

[Approved with PullApprove](https://pullapprove.com/OpenPoGo/OpenPoGoBot/pull-request/159/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=OpenPoGo/OpenPoGoBot)

**Comment by andythorne** [comment]
@OpenPoGo/maintainers I'm stuggling to find a local pokestop to test this on (dont want to jump to japan and get banned!). Can anyone else try test it?

however, i tried to run a test on the console and got this:

```
>>> "idzie wąż wąską dróżką".encode('ascii', 'replace')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
UnicodeDecodeError: 'ascii' codec can't decode byte 0xa2 in position 18: ordinal not in range(128)
```

**Comment by wchill** [comment]
Please leave a comment like `# TODO: Make this proper unicode` or something so we know to fix it later.

**Comment by Reaver01** [comment]
Commented

**Comment by Saturate** [comment]
Does not crash for me, so works as expected on windows.

**Comment by GutZuFusss** [comment]
Works fine for me on Linux aswell.

**Comment by 1karu5** [comment]
works fine here too

**Comment by MaxLeiter** [comment]
:+1: note to whoever merges: squash commits

**Comment by widnyana** [comment]
you did a great work @Reaver01, thanks!

**Comment by wchill** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/OpenPoGo/OpenPoGoBot/pull-request/159/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=OpenPoGo/OpenPoGoBot)

**Comment by kazyka** [comment]
:+1: 

[Approved with PullApprove](https://pullapprove.com/OpenPoGo/OpenPoGoBot/pull-request/159/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=OpenPoGo/OpenPoGoBot)

**Pull request merged**